homeassistant/components/tplink/light.py

```diff
@@ -99,12 +99,12 @@ async def async_setup_entry(
         raise PlatformNotReady
 
 
-def brightness_to_percentage(byt):
+def brightness_to_percentage(brightness: int) -> int:
     """Convert brightness from absolute 0..255 to percentage."""
-    return round((byt * 100.0) / 255.0)
+    return round((brightness * 100.0) / 255.0)
```

Comment:
Why is this int when all values in expression are floats?

Reply:
Round converts to an integer if a single parameter is passed.